apps/website/src/pages/docs/variables.mdx

```diff
@@ -94,12 +82,10 @@ The status colors are used for indicating the status of an element: success, war
       'var(--iui-color-background-warning-hover)',
       'var(--iui-color-background-negative-hover)',
     ],
-
-]}
+  ]}
 />
 
-Because the default status backgrounds feel so heavy and saturated, iTwinUI also offers a set of subtle background colors.
-These should be used when there is text on top of them.
+If you want to use color to help convey some sort of status, but don't want the color to be as strong or if you want to use normal text / icon colors above this background color, you can use the muted status background color palette.
```

Comment:
This sentence seemed confusing. I tried to reword the best I can, but if this isn't accurate, let me know

```suggestion
The muted status background color palette can be used to help convey some sort of status without using a strong color. It can also be used for normal text or icons above a main background color
```